src/api/resetAdminShadow.ts

```diff
@@ -0,0 +1,33 @@
+import {
+	IoTDataPlaneClient,
+	UpdateThingShadowCommand,
+} from '@aws-sdk/client-iot-data-plane'
+import { fromUtf8 } from '@aws-sdk/util-utf8-browser'
+import type { Static } from '@sinclair/typebox'
+import type { gameControllerAdminShadow } from 'api/validateGameAdminShadow'
+type GameMetadata = Static<
+	typeof gameControllerAdminShadow
+>['state']['reported']
+
+export const resetAdminShadow = async (
```

Comment:
**nitpick:** this adds [a 5th piece](https://github.com/search?q=org%3ANordicPlayground+UpdateThingShadowCommand&type=code) of code that updates the thing shadow.